src/common.h

```diff
@@ -1,17 +1,20 @@
-
 #ifndef RGRPC_H
 #define RGRPC_H
 
+#include <Rcpp.h>
+void debugLog(std::string message);
 
 /* uncomment for debugging */
 // #define RGRPC_DEBUG
 
 #ifdef RGRPC_DEBUG
-#define RGRPC_LOG(x) Rcout << x << "\n";
+#define RGRPC_LOG(x) Rcout << x << std::endl;
```

Comment:
Given that you've implemented real logging, It might be time to get rid of this macro entirely.

Reply:
I think that's fine to have even more detailed logging if someone recompiles with `RGRPC_DEBUG` -- it's not doing any harm